openfisca_france_inheritance/parameters/droits_mutation_titre_gratuit/pacte_dutreil/bonus_donation.yaml

```
@@ -0,0 +1,38 @@
+description: Bonus - Donations avant 70 ans
+values:
+  2003-08-05:
+    value: 0.5
+  2005-08-03:
+    value: 0.5
+  2007-09-26:
+    value: 0.5
+  2007-12-28:
+    value: 0.5
+  2011-07-30:
+    value: 0.5
+metadata:
+  short_label: Bonus - Donations avant 70 ans
+  last_value_still_valid_on: "2023-04-01"
```

```suggestion
  last_value_still_valid_on: "2025-06-12"
```

https://avocat-droit-succession-cahen.fr/avant-prevoir/la-transmission-du-patrimoine-professionnel-le-pacte-dutreil/